.travis.yml

```diff
@@ -7,42 +7,44 @@ branches:
   only:
     - master
 
+python:
+  - 3.6
+  - 3.7
+  - 3.8
+
+stages:
+  - check
+  - docs
+  - test
+
 matrix:
   include:
-    - python: 3.7
-      env: NOX_SESSION=check
-    - python: 3.7
-      env: NOX_SESSION=docs
-
-    - python: 3.6
-      env: NOX_SESSION=test-3.6
-    - python: 3.7
-      env: NOX_SESSION=test-3.7
-    - python: 3.8
-      env: NOX_SESSION=test-3.8
-      dist: bionic # Required to get OpenSSL 1.1.1+
-
-    - os: windows
+    -
+      stage: check
+      python: 3.7
+      script: scripts/check
+    - stage: docs
+      python: 3.7
+      script: scripts/docs
```

Comment:
Should we use `scripts/docs build` and have the script accept $@ command line arguments?